client/web/src/components/login-toggle.tsx

```diff
@@ -224,7 +229,7 @@ function LoginPopoverContent({
           ) : (
             // User is connected over tailscale, but doesn't have permission to manage.
             <p className="text-gray-500 text-xs">
-              You don’t have permission to make changes to this device, but you
+              You don't have permission to make changes to this device, but you
```

Comment:
unintentional?

Reply:
I don't think our linter catches it over here, but on the admin panel we enforce smart quotes I believe.

Reply:
oh, interesting!  I'll go look at that real quick and potentially update the other way here

Reply:
reverted this change.  I'll do a follow-up PR to add the curly-quotes linter rule and fix the web client.